src/routes/+page.svelte

```
@@ -1,2 +1,23 @@
-<h1>Welcome to Questers</h1>
-<p>Visit <a href="https://svelte.dev/docs/kit">svelte.dev/docs/kit</a> to read the documentation</p>
+<script lang="ts">
+	import Button from '$lib/components/ui/button/button.svelte';
```

ESLint

[error] 'Button' is defined but never used.

(@typescript-eslint/no-unused-vars)